include/nccl_ofi_rdma.h

```diff
@@ -328,7 +332,7 @@ typedef struct nccl_ofi_rdma_connection_info {
 	/* Message type
 	 * either NCCL_OFI_RDMA_MSG_CONN or NCCL_OFI_RDMA_MSG_CONN_RESP
 	 */
-	uint16_t type;
+	nccl_ofi_rdma_msg_type_t type;
```

Comment:
Nit: you can change this type in `nccl_net_ofi_rdma_ctrl_msg_t` as well for consistency.

Reply:
Good catch @rauteric!